.travis.yml

```diff
@@ -28,3 +28,6 @@ script:
  - ./setup.sh $HOME/wtsi-opt
  - ls -l $HOME/wtsi-opt/bin/alleleCounter
  - ls -l $HOME/wtsi-opt/bin/alleleCounter.pl
+ - echo 'Build and check docker image'
+ - docker build -t allelec .
+ - docker images | grep -c allelec
```

Comment:
Can we extend to be similar to [other tools](https://github.com/cancerit/cgpCaVEManWrapper/blob/94e00d35043e5bb43c1de57a1de5436f0a40833e/.travis.yml#L27) and ensure version is output correctly for everything?

```
docker run -t --rm allelec alleleCounter -v
docker run -t --rm allelec alleleCounter.pl -v
```